pkg/arvo/app/chat-hook.hoon

```diff
@@ -38,10 +40,11 @@
 ++  prep
   |=  old=(unit state)
   ^-  (quip move _this)
-  ?~  old
-    :_  this
-    [ost.bol %peer /permissions [our.bol %permission-store] /updates]~
-  [~ this(+<+ u.old)]
+  :_  ?~(old this this(+<+ u.old))
+  :~  (invite-poke [%create /chat])
+      [ost.bol %peer /invites [our.bol %invite-store] /invitatory/chat]
+      [ost.bol %peer /permissions [our.bol %permission-store] /updates]
+  ==
```

Comment:
Automatically subscribing on `+prep` like this could result in many duplicate subscriptions. The duct behind the bone (`ost.bol`) could be an ames packet, a key stroke, a %clay sync, a %behn timer ....

Reply:
This is surprising news to me. I will have to look into this, as it may be an issue.

Reply:
This is probably true.  The easiest way to verify it would be to start a fakezod and fake ~bud, change the app on ~bud and |merge that change to ~zod, then change the file again locally on ~zod.  I believe this will give you three separate subscriptions (one for the |start keystroke, one for the ames packet, and one for the |commit keystroke).  It is possible that they could end up on the same bone (maybe they're all really on the |start duct because that's what kicks off the ford live build?), so I'd verify before changing.

Reply:
Okay, the newest commit fixes this issue for the chat-hook. Thanks! @jtobin This should be ready to be OTA'ed.

Reply:
@loganallenc :+1: I'm going to make a hotfix today that will contain just enough changes to hopefully fix the current network problems, but will follow it up with another update containing these changes later in the week.